contracts/wallet/WalletFactory.sol

```diff
@@ -239,15 +233,64 @@ contract WalletFactory is Owned, Managed {
         //do nothing
     }
 
+    // *************** Internal Functions ********************* //
+
+    /**
+     * @dev Helper method to create a wallet for an owner account.
+     * The wallet is initialised with a list of modules, a first guardian, and an ENS..
```

Comment:
Can we remove this ".." after ENS. It's leading me to think there's something else not said there.